### ISSUE TITLE
Add ESLint and/or Prettier for more consistent code formatting

### ISSUE THREAD

**Issue by Mr0grog** [Mr0grog/google-docs-to-markdown#75]
Some of the code style here is still a bit messy as a result of this originally being an afternoon experiment. I should really add proper linting or autoformatting.

**Comment by trustedtomato** [comment]
I could take this one, I like to have prettier in my projects anyway, it would be especially important here, since multiple people contribute to the project, and I, for example, tend to forget about the semicolons.

**Comment by Mr0grog** [comment]
Sure, this would be great to have. I would like to use ESLint at least with the [`eslint-plugin-compat` plugin](https://www.npmjs.com/package/eslint-plugin-compat) to help catch usage of APIs that are too new (using Prettier for all the main formatting stuff is still fine; I agree it generally things simpler, even if it is a different code style than I’ve been using). I’ve been using the following config for `compat` locally:

```js
  "settings": {
    "browsers": "> 0.5%, last 2 versions, Firefox ESR, node >= 16, not dead, not op_mini all",
    "lintAllEsApis": true
  },
  "overrides": [
    {
      "files": ["scripts/**/*.js"],
      "settings": {
        // No browser APIs. Using some Node 18 built-ins, which is not good, but ¯\_(ツ)_/¯
        "browsers": "node >= 18"
      }
    },
    {
      "files": ["test/e2e/**/*.js"],
      "settings": {
        // No browser APIs.
        "browsers": "node >= 16"
      }
    },
    {
      "files": ["test/unit/**/*.js", "test/support/**/*.js"],
      "excludedFiles": ["test/support/wdio-webpack-dev-server.js"],
      "settings": {
        // No Node APIs.
        "browsers": "> 0.5%, last 2 versions, Firefox ESR, not dead, not op_mini all"
      }
    }
  ]
```

**Comment by Mr0grog** [comment]
Fixed in #136.
